src/systemctl/systemctl-logind.c

```diff
@@ -55,7 +57,9 @@ int logind_reboot(enum action a) {
         sd_bus *bus;
         int r;
 
-        if (a < 0 || a >= _ACTION_MAX || !actions[a])
+        assert(a >= 0 && a < _ACTION_MAX);
```

Comment:
we usually do this in two asserts. It's just a lot easier to debug if you known if the value is too low or too high. i.e. please change to:

```c
assert(a >= 0);
assert(a < _ACTION_MAX);
```